dev/format_code.sh

```diff
@@ -38,6 +40,11 @@ case $i in
   ALL="--all"
   shift
   ;;
+  --compare-against)
```

Comment:
Just as an idea, what would you think about renaming it to `--base`? In version control we usually talk about the _base_ branch (or commit) for such things so I think this would still be very intuitive and at the same time it's even shorter :)

Of course the `COMPARE_AGAINST` should then be renamed as well.

Reply:
Ah and btw, this new CLI flag should also be added to the documentation comment block at the top of the script ;)

Reply:
:-D There is documentation ? My IDE collapsed it automatically so I didn't realized there is one ... :facepalm: 

the `--base` wording is definitely better so I use that